src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,20 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        Figure[] array = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
 
+        for (int i = 0; i <= 2; i++) {
```

Comment:
Consider changing the loop condition to `i < 3` instead of `i <= 2` for better readability and consistency with the second loop.